pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
```

Comment:
Seems like the interface should better describe what it's able to do.  Perhaps splitting the client interfaces into read-only interfaces and read-write interfaces would allow this to provide useful interfaces.